src/ImGui/Desktop/Fusee.ImGui.Desktop/Templates/ImGuiFilePicker.cs

```diff
@@ -461,33 +440,27 @@ public virtual unsafe void Draw(ref bool filePickerOpen)
             return;
         }
 
-        private List<string> GetFileSystemEntries(string fullName)
+        private List<FileSystemInfo> GetFileSystemEntries(string fullName)
         {
             try
             {
-                var files = new List<string>();
-                var dirs = new List<string>();
+                var folders = new List<DirectoryInfo>();
+                var files = new List<FileSystemInfo>();
```

Comment:
This seems to be inconsistent:
`FileSystemInfo` is the base of both `DirectoryInfo` and `FileInfo`.
So in line 448 it should be `FileInfo`?

Reply:
Yes. Same base class, however this should be more consistent.